doc/examples/filters/plot_hysteresis.py

```diff
@@ -0,0 +1,50 @@
+"""
+=======================
+Hysteresis thresholding
```

Comment:
Should probably add this method also to http://scikit-image.org/docs/dev/auto_examples/xx_applications/plot_thresholding.html .